foyer/forcefields/xml/oplsaa.xml

```diff
@@ -220,25 +220,25 @@
   <Type name="opls_215" class="CT" element="C" mass="12.011" def="[C;X4]([S;%opls_203])(H)" desc="C in 2R-CH-S-S-*"  overrides="opls_216, opls_211" doi="10.1021/ja9621760"/>
   <Type name="opls_216" class="CT" element="C" mass="12.011" def="[C;X4][S;%opls_203]" desc="C in 3R-C-S-S-*" overrides="opls_212" doi="10.1021/ja9621760"/>
   <Type name="opls_217" class="CT" element="C" mass="12.011" def="[C;X4]([S;%opls_200])(H)(H)(H)" desc="C in CH3-S-H (methanethiol)" overrides="opls_211, opls_207, opls_209, opls_206" doi="10.1021/ja9621760"/>
-  <Type name="opls_218" class="opls_218" element="C" mass="12.011"/>
+  <Type name="opls_218" class="CT" element="C" mass="12.011" def="[C;X4]([C;%opls_221])(O)(H)(H)" overrides="opls_149, opls_157, opls_158"/>
   <Type name="opls_219" class="opls_219" element="C" mass="12.011"/>
   <Type name="opls_220" class="opls_220" element="C" mass="12.011"/>
-  <Type name="opls_221" class="opls_221" element="C" mass="12.011"/>
+  <Type name="opls_221" class="CA" element="C" mass="12.011" def="[C;X3;r6]([C;X3;r6])([C;X3;r6])([C;X4](O)(H)(H))" overrides="opls_145"/>
   <Type name="opls_222" class="opls_222" element="S" mass="32.06"/>
   <Type name="opls_223" class="opls_223" element="C" mass="12.011"/>
   <Type name="opls_223B" class="opls_223B" element="C" mass="12.011"/>
   <Type name="opls_224" class="opls_224" element="C" mass="12.011"/>
   <Type name="opls_224B" class="opls_224B" element="C" mass="12.011"/>
   <Type name="opls_225" class="opls_225" element="C" mass="12.011"/>
   <Type name="opls_225B" class="opls_225B" element="C" mass="12.011"/>
-  <Type name="opls_226" class="opls_226" element="Cl" mass="35.453"/>
-  <Type name="opls_227" class="opls_227" element="C" mass="12.011"/>
+  <Type name="opls_226" class="Cl" element="Cl" mass="35.453" def="[Cl;X1]([C;%opls_227])" desc="chloroalkene Cl" overrides="opls_401" doi="10.1021/ct200731v"/>
+  <Type name="opls_227" class="CM" element="C" mass="12.011" def="[C;X3;!r6]([C;X3;!r6](H)(H))(Cl)(Cl)" desc="chloroalkene C (ClH-C=)" doi="10.1021/ct200731v"/>
   <Type name="opls_228" class="opls_228" element="C" mass="12.011"/>
   <Type name="opls_229" class="opls_229" element="C" mass="12.011"/>
   <Type name="opls_230" class="opls_230" element="C" mass="12.011"/>
   <Type name="opls_231" class="opls_231" element="C" mass="12.011"/>
-  <Type name="opls_232" class="opls_232" element="C" mass="12.011"/>
-  <Type name="opls_233" class="opls_233" element="C" mass="12.011"/>
+  <Type name="opls_232" class="C_2" element="C" mass="12.011" def="[C;X3]([O;X1])([C;X3])[H]" desc="Carbon in C=0 group of benzaldehyde,acetophenone" overrides="opls_277" doi="10.1021/ct200731v"/>
```

Comment:
The only difference between these two (232 vs 233) should be the H vs C. So the SMARTS should look like
`[C;X3]([O;X1])([c;X3&r6])(C)` and `[C;X3]([O;X1])([c;X3&r6])(H)`

Reply:
Good catch. Fixed.